web/views/coherence/coherence_view_helpers.ex

```diff
@@ -37,15 +38,17 @@ defmodule Ex338.Coherence.ViewHelpers do
   """
   def coherence_links(conn, which, opts \\ [])
   def coherence_links(conn, :new_session, opts) do
-    recover?  = Keyword.get opts, :recover, @recover_link
-    unlock?   = Keyword.get opts, :unlock, @unlock_link
-    register? = Keyword.get opts, :register, @register_link
+    recover_link  = Keyword.get opts, :recover, @recover_link
+    unlock_link   = Keyword.get opts, :unlock, @unlock_link
+    register_link = Keyword.get opts, :register, @register_link
```

Comment:
Assigned variable `register_link` has same name as a function in the same module.